src/scss/_mixins.scss

```diff
@@ -80,11 +80,15 @@
 		display: block;
 	}
 
-	// Standardise outline styles
-	:focus {
-		outline: 2px solid $o-normalise-focus-color;
+	// Standardise focus styles. This is scoped because IE allows the body tag to have focus which is... odd
```

Comment:
Another approach could be `body *:focus`, that would mean anything focused which is a descendant of the body node.

Reply:
The thing I'm concerned about is IE giving focus to other odd things, this seemed like a safer way to work around this problem.
Are you concerned we might miss other focussable element types?

Reply:
That is my concern, yes, other focusable elements which don't require a tabindex attribute are: input, select, textarea, object, area;

Another alternative would be to keep the root `:focus` styling and override it for the html, body and readonly nodes:
```
html:focus, body:focus, input[readonly]:focus {
    outline: none
}
```

I saw that approach in the default stylesheet bundled within chromium. https://cs.chromium.org/chromium/src/third_party/WebKit/Source/core/css/html.css?l=990

Reply:
Yeah I agree with using something like the following, after properly giving it some thought:

```
html:focus, body:focus, input[readonly]:focus {
    outline: none
}```

It's better for some elements to have focus when we don't want them to than the other way around, which would present a11y issues. So yep I reckon go with the blacklist approach Jake suggested.

Reply:
thanks team!